CMakeLists.txt

```diff
@@ -162,3 +177,12 @@ install (FILES
     ${WAMR_ROOT_DIR}/core/iwasm/include/wasm_export.h
     ${WAMR_ROOT_DIR}/core/iwasm/include/lib_export.h
     DESTINATION include)
+
+# If it's a Release build, strip the shared library
+if (CMAKE_BUILD_TYPE STREQUAL "Release")
```

Comment:
Same as above, had better also check CMAKE_STRIP?

Reply:
Yes, I added `CMAKE_STRIP` check both conditions that I also check Release type.